### PR TITLE
Changes API call for incident for recursive fetch from Gurucul

### PR DESCRIPTION
**Pull request by gra-xsoar**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Gurucul/ReleaseNotes/1_1_2.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#### Integrations
+##### Gurucul-GRA
+
+- **Bug Fixes**:
+  - **Breaking Change**: Fixed an issue with fetch incidents.
```
Comment (Shellyber) [inline Packs/Gurucul/ReleaseNotes/1_1_2.md:5]: @gra-xsoar Why the breaking changes?

Comment (gra-xsoar) [reply Packs/Gurucul/ReleaseNotes/1_1_2.md:5]: its incident fetch change that's why we wrote: "Breaking change"?
if not required then we will update the release file

**Comment by Shellyber** [comment]
@gra-xsoar Can you please explain what is the change? What is the difference between then and the fix now? 
Also please update the docker image of the integration to `3.9.6.22912` and add this update to the release notes as well

**Comment by gra-xsoar** [comment]
1. Prior we provide a call to our API per minute which by default set in Demisto and the same way we implemented in our API but client-side default time was set as 60 min and for the same reason our code is not synced up with demisto data
so for the quick fix, we removed our per minute API call and now we gave the call for the whole day, and for avoiding duplicates we are given them as pre-process rule.in the next release, we will fix this issue which is independent of the duration.

2. I updated the docker image as 3.9.6.22912 in integration.
3. Removed breaking change

**Review by Shellyber — APPROVED** [review]

**Pull request merged**